src/reporters/composite_reporter.js

```diff
@@ -33,24 +33,30 @@ export default class CompositeReporter {
     }
 
     report(span: Span): void {
-        for (let i = 0; i < this._reporters.length; i++) {
-            let reporter = this._reporters[i];
-            reporter.report(span);
-        }
+        this._reporters.forEach((r) => {
+            r.report(span);
+        });
+    }
+
+    flush(callback: ?Function): void {
+        this._reporters.forEach((r) => {
+            r.flush(callback);
```

Comment:
I thought we didn't need that.  If you want to assert how may times something is called then sinon has that.